README.md

```diff
@@ -66,16 +66,16 @@ require 'path/to/PHPMailer/src/PHPMailer.php';
 require 'path/to/PHPMailer/src/SMTP.php';
 ```
 
-If you're not using the `SMTP` class explicitly (you're probably not), you don't need a `use` line for the SMTP class.
+If you're not using the `SMTP` class explicitly (you're probably not), you don't need a `require` line for the SMTP class.
```

Comment:
The original is correct here. You always need a `require` line for the SMTP class if you're using SMTP (without composer), but you don't need a `use` line unless your'e using it directly, i.e. creating your own SMTP instances.